continuous_integration/kubernetes/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+echo "Deploying chartpress..."
+
+set -eu
+# TODO: add deploy-key.rsa.enc and set keys in travis UI
+openssl aes-256-cbc -K $encrypted_058f1ad78f7f_key -iv $encrypted_058f1ad78f7f_iv -in deploy-key.rsa.enc -out deploy-key.rsa -d
```

Comment:
@jcrist - this is the main thing you'll need to do.

Reply:
Any reference on what I need to do here to make this work?

Reply:
@jacobtomlinson setup our pangeo chart: https://github.com/pangeo-data/helm-chart/pull/1

In travis CI, we set 4 env variables:

![image](https://user-images.githubusercontent.com/2443309/66416583-fecfea00-e9b2-11e9-8347-21fea623a2ff.png)

Reply:
@jcrist do you mind if I do this? I don't think it matters who (if anyone) has the unencrypted deploy key.

Reply:
I was just looking into this. I'll take care of it, I want to make sure it integrates nicely with our CI setup (which is a bit complicated).

Reply:
You'll want to generate an ssh key for Travis to use to deploy.

```bash
ssh-keygen -t rsa -b 4096 -q -N "" -C "" -f continuous_integration/kubernetes/deploy-key.rsa
```

You'll need to create a deploy key over on https://github.com/dask/helm-chart with the contents of the `deploy-key.rsa.pub` file that gets generated.

Then use the Travis CI CLI tool to encrypt it. It will use openssl to encrypt it and automatically add the decryption keys to Travis CI as environment variables.

```bash
travis encrypt-file continuous_integration/kubernetes/deploy-key.rsa --com --add

# You might need to move the encrypted file to the correct location as it will probably
# just drop it in your working directory
mv deploy-key.rsa.enc continuous_integration/kubernetes/deploy-key.rsa.enc
```

You may also want to add the `deploy-key.rsa` and `deploy-key.rsa.pub` files to the `.gitignore` for safety.

Lastly the Travis CI CLI command will print out the openssl line that you need to replace this one with here. Again the path to the `deploy-key.rsa.enc` might be wrong and need changing to `continuous_integration/kubernetes/deploy-key.rsa.enc`.